packages/params-middleware/src/middleware.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import { RSAA } from 'redux-api-middleware';
+
+import type { Dispatch } from 'redux';
+import type { Action } from './types';
+
+function parametriseEndpoint({ endpoint, types }) {
+  const params = types[0]?.meta?.params;
```

Comment:
What do you think about adding `params` directly on the same level as `endpoint`, `body` etc?
This was my version of this middleware:
```
  const apiCall = action[RSAA];

  if (!apiCall || !apiCall.params) {
    return next(action);
  }

  const { endpoint, params, ...other } = apiCall;

  return next({
    [RSAA]: {
      ...other,
      endpoint: `${endpoint}?${stringify(params)}`,
    },
  });
```

Reply:
And if you were thinking about convenience of having the params in the related actions,
then I would say it should be present in all of them and not only in the request.